src/cards/slow.ts

```diff
@@ -50,20 +59,23 @@ const spell: Spell = {
     category: CardCategory.Curses,
     sfx: '',
     supportQuantity: true,
-    manaCost: 20,
+    manaCost: 15,
     healthCost: 0,
     expenseScaling: 1,
     probability: probabilityMap[CardRarity.UNCOMMON],
     thumbnail: 'spellIconSlow.png',
     animationPath: '',
-    description: ['spell_slow', Math.floor(changeProportion * 100).toString()],
+    description: ['spell_slow', Math.floor(100 - slowPercentage).toString()],
     effect: async (state, card, quantity, underworld, prediction) => {
       // .filter: only target living units
       const targets = state.targetedUnits.filter(u => u.alive);
       if (targets.length) {
         await Promise.all([playDefaultSpellAnimation(card, targets, prediction), playDefaultSpellSFX(card, prediction)]);
         for (let unit of targets) {
-          Unit.addModifier(unit, slowCardId, underworld, prediction, quantity);
+          // Each application of slow is multiplicative, so we must add the modifier like this
+          for (let i = 0; i < quantity; i++) {
+            Unit.addModifier(unit, slowCardId, underworld, prediction, slowPercentage);
+          }
```

Comment:
Why not just pass the quantity to addModifier and do the loop inside the add function?
Would that interfere with other applications of slow like in poison or could moving the loop to the slow's add function mean we don't need both loops elsewhere?

Reply:
Slows are multiplicative, so applying 3 50% slows and applying a 150% slow do different things

Heavy toxins is written this way so that the rune can stack, without giving players the ability to apply slows at or over 100%.

We can't do it in the slow function either, because applying 20 1% slows is different than applying a 20% slow

50% slow + 50% slow = 75% slow
90% slow + 50% slow = 95% slow